modules/local/bedtobedgraph_bystrand.nf

```diff
@@ -0,0 +1,34 @@
+process BEDTOBEDGRAPH_BYSTRAND {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "onda-forge::gawk=5.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
+        'quay.io/biocontainers/gawk:5.1.0' }"
+
+    input:
+    tuple val(meta), path(ctss)
+
+    output:
+    tuple val(meta), path("*.pos.bedgraph") , emit: pos_bedgraph
+    tuple val(meta), path("*.neg.bedgraph") , emit: neg_bedgraph
+    path  "versions.yml"                , emit: versions
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+
+    awk '\$6=="+"' $ctss > ${prefix}.pos.bed
```

Comment:
not sure about this module - its back to doing multiple things in one place. This could be broken out into multiple awk processes, is there a specific design reason to keep these operations together?